### PR TITLE
Add AfriGIS as an Experienced Provider

### PR DESCRIPTION
**Pull request by petersmythe**

### CODE REVIEW

File: support/index.html [hunk 1]
```diff
@@ -166,6 +166,30 @@ <h2>Experienced Providers</h2>
...
+        </p></div>
+        <div class="col-xs-10">
+            <ul class="experience">
+               <li>training</li>
+               <li>support</li>
+               <li>development</li>
+               <li>documentation</li>
+               <li>committer</li>
+               <li>service</li>
+               <li>product</li>
```
Comment (jodygarnett) [inline support/index.html:188]: What is your product that uses geoserver?

File: support/index.html [hunk 2]
```diff
@@ -166,6 +166,30 @@ <h2>Experienced Providers</h2>
...
+        <div class="col-xs-10"><p>
+            <a href="https://www.afrigis.co.za/">AfriGIS</a> (South Africa) has been providing geospatial data in Africa since 1997. We specialise in international standards compliant location-sensitive data and solutions. We provide web tools and <a href="https://developers.afrigis.co.za">APIs</a> to help customers connect to, enhance and enrich their data with location information.
+</p><p>
+			AfriGIS operates and manages many GeoServer instances in our own and client cloud environments. We also provide custom development, training and support for GeoServer and QGIS. We believe that big businesses should help open-source projects in any way they can. This includes providing funding, writing documentation and custom plugins, offering support, and fixing issues.
+</p><p>
+			Visit our website for more of our geospatial data technology or <a href="https://www.afrigis.co.za/contact-us/">contact us</a> for support.
+        </p></div>
+        <div class="col-xs-10">
+            <ul class="experience">
+               <li>training</li>
```
Comment (jodygarnett) [inline support/index.html:182]: I just had the cool idea that we could make these into links :)

**Review by jodygarnett — APPROVED** [review]
I had some questions out of curiosity;  it the PR looks good. 

Thank you for supporting geoserver.

File: support/index.html (hunk 1 above)
Comment (petersmythe) [reply support/index.html:188]: Sorry, internally we call it a product, but it is actually more of a service.  I will remove.

File: support/index.html (hunk 2 above)
Comment (petersmythe) [reply support/index.html:182]: I will try to do that over time

Comment (jodygarnett) [reply support/index.html:182]: Do you think it will look good?

**Pull request merged**